tool/state-processor/src/adjust.rs

```diff
@@ -35,11 +35,21 @@ impl Adjust for BalanceLock {
 
 impl Adjust for VestingInfo {
 	fn adjust(&mut self) {
+		let now = *NOW.read().unwrap();
+		let released = (now - self.starting_block) as Balance * self.per_block;
+
+		if self.locked <= released {
+			self.locked = 0;
+
+			return;
+		} else {
+			self.locked -= released;
+		}
+
 		self.locked.adjust();
-		self.per_block *= 2;
 		self.per_block.adjust();
-		self.starting_block =
-			self.starting_block.checked_sub(*NOW.read().unwrap()).unwrap_or_default();
+		self.per_block *= 2;
```

Comment:
The vesting module unlocked balance as the block number increase. 
This process is linked to the block number rather than timestamp. So I think we don't need to double this value.

Reply:
I don't think so.

For example, our team's vesting plan is to unlock all team reservations in 2024. Without this modification, the time will be 2026.

Reply:
Thanks for your reply. This reason make sense to me.